Source/WebKit/WebProcess/Extensions/API/Cocoa/WebExtensionAPIWebPageNamespaceCocoa.mm

```diff
@@ -67,7 +69,7 @@
     // Documentation: None (Testing Only)
 
     if (!m_test)
-        m_test = WebExtensionAPITest::create(contentWorldType());
+        m_test = WebExtensionAPITest::create(*this);
```

Comment:
Can you explain this change?

Reply:
This makes it more like all the other APIs, which infers its defaults from the patron object (`this`). Only `runtime` needs to be special because it has some custom intricacies.